src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/share/flow/resources/CookiePoolTest.java

```diff
@@ -35,26 +39,16 @@ public void setUp() {
 
     @Test
     public void cookiePool() {
-        long first = cookiePool.allocate("flow_1");
-        assertEquals(5, first);
-
-        long second = cookiePool.allocate("flow_2");
-        assertEquals(6, second);
-
-        long third = cookiePool.allocate("flow_3");
-        assertEquals(7, third);
-
-        cookiePool.deallocate(second);
-        long fourth = cookiePool.allocate("flow_4");
-        assertEquals(6, fourth);
-
-        long fifth = cookiePool.allocate("flow_5");
-        assertEquals(8, fifth);
+        Set<Long> cookies = new HashSet<>();
+        for (int i = 5; i <= 25; i++) {
+            cookies.add(cookiePool.allocate(format("flow_%d", i)));
```

Comment:
The test became shallow: it doesn't verify the allocated values and doesn't check deallocation. I think additional tests should be added to cover this.

Reply:
Tests have been added.